src/PowerShellEditorServices/Session/RunspaceDetails.cs

```diff
@@ -229,7 +229,7 @@ internal bool HasCapability<TCapability>()
                 // Attaching to process currently needs to be marked as a local session
                 // so we skip this if block if the runspace is from Enter-PSHostProcess
                 if (hostName.Equals("ServerRemoteHost", StringComparison.Ordinal)
-                    && !(runspace.OriginalConnectionInfo is NamedPipeConnectionInfo))
+                    && !(runspace.OriginalConnectionInfo.GetType().ToString() == "NamedPipeConnectionInfo"))
```

Comment:
This might be a tad easier for humans to parse as:
```
… && (runspace.OriginalConnectionInfo.GetType().ToString() != "NamedPipeConnectionInfo"))
```

Reply:
yep :) tunnel vision after the CI failure because of this type not being in PSv3

Reply:
```suggestion
                    && runspace.OriginalConnectionInfo.GetType().ToString() == "NamedPipeConnectionInfo")
```

Reply:
Has happened to me more than I'd care to admit. :-)